episode/urls.py

```diff
@@ -2,5 +2,6 @@
 from episode import views
 
 urlpatterns = [
-    url(r'^$', views.home)
+    url(r'^$', views.home),
+    url(r'^about', views.about)
```

Comment:
I personally like to have the more specific things up top and more general things on the bottom